61.md

```diff
@@ -0,0 +1,76 @@
+NIP-61
+======
+
+Unbound List
+--------------
+
+`draft` `optional`
+
+An unbound list is a group of events with the same pubkey and `n` (list "n"ame) tag.
+An event can have many `n` tags, thus being member of many unbound lists.
+
+To add an event to the "Close Friends" list,
+add a "close-friend" (prefer lowercase English words in singular form with dash separator) `n` tag to the event.
+
+To remove an event from a "Close Friends" list,
+remove the "close-friend" `n` tag from the event.
+
+An unbound list itself is referenced by an `u` tag
+or by an `nlist` [NIP-19](19.md) entity using [NIP-21](21.md) URI.
+
+## Use Cases
+
+Unbound lists can be used wherever a [NIP-51](51.md) list fits.
+
+NIP-51 lists have a limit on the number of items they can hold depending on relay's event size restrictions.
+While unbound lists can have any number of items.
+
+The downside of unbound lists is that migrating a list from one relay to another can be rate limited
+due to saving many individual events during a small time window.
+
+Public and encrypted data can coexist in unbound list events, similar to NIP-51 lists.
+
+## Referencing
+
+An unbound list is referenced by an `u` tag.
+It references a set of events from an author instead of a single one:
+
+`["u", "<32-bytes lowercase hex of a pubkey>:<n tag value>", "<recommended relay URL, optional>"]`
```

Comment:
I would keep the kind there `kind:pubkey:n` but allow kind to be empty for lists from multiple event kinds (this seems like a rare case).